scripts/packages/msi/make_msi_lib.ps1

```diff
@@ -26,8 +26,8 @@ function Log-Info {
       [string]$msg
     )
     $basename = $PSCommandPath.Substring($PSCommandPath.LastIndexOfAny('/\') + 1)
-    $time = (Get-Date).ToString(.HH:mm:ss.)
```

Comment:
Just curious, what does `.<string>.` mean in Powershell?

Reply:
Nothing, it was a mistake. I don't know how I made it...